src/components/Header.js

```diff
@@ -91,6 +91,8 @@ class Header extends React.Component {
     const styles = reactCSS(stylesheet, {
       'mobile': isMobile
     });
+    if(this.props.color) styles.container.backgroundColor = this.props.color;
+
```

Comment:
Don't do this. style concerns should not have their own prop. instead do something like:

```
<Header style={{ backgroundColor: 'black' }} />
```

Then in Header.jsx:
```
<div style={{ ...styles.container, ...this.props.style }} />
```